readthedocs/builds/managers.py

```diff
@@ -219,9 +218,8 @@ def add_rule(
             priority=priority,
             description=description,
             match_arg=match_arg,
-            predefined_match_arg=predefined_match_arg,
             version_type=version_type,
             action=action,
-            action_arg=action_arg,
```

Comment:
Why we removed these? Seems clearer to use named attributes.

Reply:
Those already have defaults, feels unnecessary having to pass every attribute, even the ones with default values.